### ISSUE TITLE
Key press immediately exits window when in OS X Fullscreen Mode

### ISSUE THREAD

**Issue by kaffeeundsalz** [quodlibet/quodlibet#1792]
In OS X, when you take Quod Libet to native fullscreen mode via the green maximize button in the title bar, any key press will immediately exit the Quod Libet window and bring you back to the desktop.

This way, it's difficult to select multiple tracks via `Shift`. Execution of keyboard shortcuts is still possible because you won't lose app focus, but you'll only be back in the Quod Libet window when the shortcut invokes a new window. For example, `Cmd+,` will work, but you have to switch back to Quod Libet manually afterwards. `Cmd+O` will indeed bring you back but still comes with a short fullscreen focus loss.

You can prevent this behaviour by selecting certain elements in the Quod Libet window, e.g. the search field or the top panes in Paned Browser view.

**Comment by lazka** [comment]
Does El Capitan do anything different compared to previous versions or is this just the window zoom which increases the window size to fill the desktop?

**Comment by kaffeeundsalz** [comment]
It's probably not an El Capitan issue, because Apple has changed virtually nothing about how window management works in 10.11. The native OS X fullscreen mode was introduced with OS X 10.7 Lion in 2011. Ever since then, there's a difference between taking an application fullscreen and just maximizing the application window to the desktop boundaries.

Essentially, what fullscreen mode does is move the main application window to its own separate space (= virtual desktop) so it occupies the entire screen, hiding the top menu bar, the window titlebar and the Dock.

There used to be separated buttons for fullscreen mode and maximize window at first, but since OS X 10.10 Yosemite, the green maximize button defaults to fullscreen mode and can be turned into a maximize button by alt-clicking it.

All of this is a bit difficult to explain if you're unfamiliar with the whole Mac terminology or haven't seen fullscreen mode in action. I found a short Youtube video that shows the feature in action, but I might as well clarify this with screenshots or a short screen recording if you wish.

**Comment by lazka** [comment]
Ah, so previously (I only have 10.9 here I think) there was this button: https://support.apple.com/kb/PH11359?locale=en_US and now the green button does the same thing by default?

I'll check it out, thanks

**Comment by kaffeeundsalz** [comment]
Exactly. The button you linked is gone since 10.10, its function merged into the green button at the left of the window titlebar. Sorry for the confusion.

**Comment by urielz** [comment]
I can't reproduce this on QL 3.8.1 and OS X 10.11.6. Anyone else that can test this on OS X?
